app/models/iam_user_identity.rb

```diff
@@ -5,6 +5,9 @@
 # introspect endpoint.Adds IAM sourced versions of ICN, EDIPI, and SEC ID to pass to the IAMUser model.
 #
 class IAMUserIdentity < ::UserIdentity
+  PREMIUM_LOAS = [2, 3].freeze
+  UPGRADE_AUTH_TYPES = %w[DSL MHV].freeze
```

Comment:
Can these values ever come to us in lower or mixed case?

Reply:
I'd need to double check with the IAM team, but I don't think so

Reply:
Anything's possible but I really doubt it.